### ISSUE TITLE
Batt value available in SPECIAL FUNCTIONS but not to LUA script

### ISSUE THREAD

**Issue by NorthIceMan** [opentx/opentx#8095]
The issue tracker here is for devlopment of OpenTX. This is not a
support site, only a bug tracking system. Please use support groups
like [rcgroups](https://rcgroups.com/) or
[openrcforum](https://www.openrcforums.com/forum/). See also our
[manual](https://opentx.gitbooks.io/manual-for-opentx-2-2/).

For quick questions you can also join our general chat.

Issues that fall into the category of asking for help, will be
generally closed as invalid without an answer.
Tried to get Tx battery voltage in LUA script using "Batt", "TxBatt" but returns 0 (zero). This when Telemetry is off. (Some folks dont use telemetry!

**Comment by NorthIceMan** [comment]
Tried to get Tx battery voltage in LUA script using "Batt", "TxBatt" but returns 0 (zero). This when Telemetry is off. (Some folks dont use telemetry!

**Comment by NorthIceMan** [comment]
OK-Problem is solvable!  After looking for days I FOUND the elusive name for the Tx Batt voltage in GitBooks online!   The now famous name, which was under the Taranis Qx7 sect is tx-voltage  .  Tried a script and it works!
Some "cleanup to do" as we used to say in programming back in the 1960's...why is "Batt" showing true voltage in Special Functions but not when accessed by LUA?  SOOO many things to do!
